end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,1834 @@
+{
```

Comment:
<p>Would you explain more about how you "synthetically" generated the dataset?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Also, is the data already normalized?</p>